libqtile/widget/base.py

```diff
@@ -29,30 +29,38 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from .. import command, bar, configurable, drawer
+from .. import command, bar, configurable, drawer, confreader
 import six
 import subprocess
 import logging
 import threading
 import warnings
 
+ORIENTATION_NONE = 0
+ORIENTATION_HORIZONTAL = 1
+ORIENTATION_VERTICAL = 2
+ORIENTATION_BOTH = 3
```

Comment:
Actually I took inspiration from `bar.STRETCH`, `bar.CALCULATED` and `bar.STATIC`, which aren't in libqtile.config either. The `base.ORIENTATION_*` bits are only needed by widget developers to define what orientation their widget supports when they instantiate it, they aren't supposed to be configured by end users... or didn't I get what you meant?